static/js/public/core.js

```diff
@@ -0,0 +1,291 @@
+/* eslint-disable no-undef, indent, no-console, no-undef */
```

Comment:
do you mean the file? nicholae wrote it (or it was included with the newer version of the template anyway).